source/v2/greenlanes-openapi.yaml

```diff
@@ -68,59 +72,58 @@ info:
     <% end %>
 
 paths:
-  /green_lanes/categorisations:
+  /green_lanes/category_assessments:
     get:
-      summary: Returns a list of all possible categorisations
+      summary: Returns a list of all category assessments
       description: >
-        Returns a list of the all possible categorisations
+        Returns a list of the all category assessments
       tags:
         - Green Lanes
       parameters:
         - name: as_of
           in: query
           required: false
           description: |
-            Returns the possible categorisations as they existed on the `as_of` date. Use the format `YYYY-MM-DD`
+            Returns the category assessments across all commodities as they existed on the `as_of` date. Use the format `YYYY-MM-DD`
 
-            While this is not a required field it is advised to include it with all requests even
-            if requesting data for today to ensure the data returned is correct. As caching is used
-            on the API this will ensure the data returned is as expected.
+            While `as_of` is not a required field it is advised to include it with all requests even
+            if requesting data for today to ensure the data returned is correct.
           schema:
             type: string
             format: date
       responses:
         200:
-          description: An array of Categorisations
+          description: An array of CategoryAssessment's
```

Comment:
Do we need the **'** ?
or is that just a plural (many CategoryAssessment) ?
`CategoryAssessments`